be/src/vec/exec/vset_operation_node.h

```diff
@@ -114,23 +114,36 @@ void VSetOperationNode::refresh_hash_table() {
 
                     arg.init_once();
                     auto& iter = arg.iter;
-                    for (; iter != arg.hash_table.end(); ++iter) {
+                    auto iter_end = arg.hash_table.end();
+                    decltype(arg.iter) iter_prev;
```

Comment:
Maybe we can just declare `iter_prev` at inner loop.

Reply:
Done